### PR TITLE
Update annotations of Serialization

### PR DESCRIPTION
**Pull request by GrimpEreau**

### CODE REVIEW

File: core/serialization.md [hunk 1]
```diff
@@ -175,17 +171,15 @@ namespace App\Entity;
...
- *         }
- *     }
- * )
- */
+#[ApiResource(
+    normalizationContext: ['groups' => ['get']],
+    itemOperations: [
+        'get',
+        'put' => [
+            'normalization_context' => ['groups' => ['put']]
```
Comment (alanpoulain) [inline core/serialization.md:179]:
```suggestion
            'normalization_context' => ['groups' => ['put']],
```

File: core/serialization.md [hunk 2]
```diff
@@ -352,16 +340,14 @@ namespace App\Entity;
...
- *      "groups" = {"person"}
- *   },
- *   denormalizationContext = {
- *      "groups" = {"person"}
- *   }
- * )
- */
+#[ApiResource(
+    normalizationContext: [
+        'groups' => ['person']
```
Comment (alanpoulain) [inline core/serialization.md:345]:
```suggestion
        'groups' => ['person'],
```

File: core/serialization.md [hunk 3]
```diff
@@ -352,16 +340,14 @@ namespace App\Entity;
...
+    ],
+    denormalizationContext: [
+        'groups' => ['person']
```
Comment (alanpoulain) [inline core/serialization.md:348]:
```suggestion
        'groups' => ['person'],
```

File: core/serialization.md [hunk 4]
```diff
@@ -394,16 +380,14 @@ namespace App\Entity;
...
- *      "groups" = {"person"}
- *   },
- *   denormalizationContext = {
- *      "groups" = {"person"}
- *   }
- * )
- */
+#[ApiResource(
+    normalizationContext: [
+        'groups' => ['person']
```
Comment (alanpoulain) [inline core/serialization.md:385]:
```suggestion
        'groups' => ['person'],
```

File: core/serialization.md [hunk 5]
```diff
@@ -394,16 +380,14 @@ namespace App\Entity;
...
+    ],
+    denormalizationContext: [
+        'groups' => ['person']
```
Comment (alanpoulain) [inline core/serialization.md:388]:
```suggestion
        'groups' => ['person'],
```

File: core/serialization.md [hunk 6]
```diff
@@ -440,13 +424,13 @@ use Doctrine\ORM\Mapping as ORM;
...
- * @ApiResource(
- *     collectionOperations={
- *          "get"={"normalization_context"={"groups"="greeting:collection:get"}},
- *     }
- * )
  * @ORM\Entity
  */
+#[ApiResource(
+    collectionOperations: [
+        'get' => ['normalization_context' => ['groups' => 'greeting:collection:get']]
```
Comment (alanpoulain) [inline core/serialization.md:431]:
```suggestion
        'get' => ['normalization_context' => ['groups' => 'greeting:collection:get']],
```

**Review by dunglas — APPROVED** [review]

**Comment by dunglas** [comment]
Thanks @GrimpEreau !

**Pull request merged**